runtime/doc/usr_10.txt

```diff
@@ -625,7 +626,7 @@ When you are typing plain text, it's nice if the length of each line is
 automatically trimmed to fit in the window.  To make this happen while
 inserting text, set the 'textwidth' option: >
 
-	:set textwidth=72
+	:set textwidth=78
```

Comment:
The paragraph below references that value being set from the example vimrc, but the example vimrc sets `78`.